tests/unit_tests/model/test_model_parameter.py

```diff
@@ -162,11 +162,11 @@ def test_load_parameters_from_db(telescope_model_lst, mocker):
     telescope_copy = copy.deepcopy(telescope_model_lst)
     mock_db = mocker.patch.object(DatabaseHandler, "get_model_parameters")
     telescope_copy._load_parameters_from_db()
-    assert mock_db.call_count == 2
+    assert mock_db.call_count == 4
```

Comment:
Is this a bad sign that we went up to 4?

Reply:
No, this is expected - we call `get_model_parameters` twice in `_load_parameters_from_db()` (for `telescope` and `sites` parameter)